jvm/common/src/main/proto/squareup/cash/s2dk/api/alpha/public.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto2";
 package squareup.cash.s2dk.api.alpha;
 
-option java_package = "com.squareup.protos.cash.deviceintegritly.api.alpha";
+option java_package = "com.squareup.protos.cash.s2dk.api.alpha";
```

Comment:
Overall question should the package be s2dk or trifle

Reply:
should be trifle, but I want to go through and clean that up separately (keeping these ports as close to original as possible within reason)